pkg/controller/replication_controller.go

```diff
@@ -60,6 +60,8 @@ type RealPodControl struct {
 // Time period of main replication controller sync loop
 const DefaultSyncPeriod = 10 * time.Second
 
+const RCNAME_ANNOTATION = "replicationControllerName"
```

Comment:
Do you care more about the replication controller, or the pod template used to create the pod?

Any annotation created by the system should be namespaced. I believe we've used `kubernetes.io/` elsewhere.